src/screens/TripDetails/components/TripSection.tsx

```diff
@@ -168,7 +172,25 @@ const TripSection: React.FC<TripSectionProps> = ({
             />
           </TripRow>
         )}
-
+        {lastPassedStop?.quay?.name && (
+          <TripRow>
```

Comment:
Should this be clickable? Kinda hard to know from the sketches. It seems kinda "grouped together" with the other clickable text like the "3 intermediate stops".

Reply:
I know we have talked about this before, that some people have a hard time finding the part that is clickable. I feel like we should make the line name clickable as well. Maybe @ckbilstad have an opinion? (Which part of a leg in trip details should be clickable, and lead to departure details?)

Reply:
I agree, it's hard to find. Maybe adding an arrow or something would help as well.

Reply:
I agree. Dont have the time to jump into it right now tho. Mention it to Sebastian or create an issue so we can pick it up again soon.

Reply:
https://github.com/AtB-AS/kundevendt/issues/3066